examples/js/loaders/TDSLoader.js

```diff
@@ -272,6 +272,12 @@ THREE.TDSLoader.prototype = Object.assign( Object.create( THREE.Loader.prototype
 				material.shininess = shininess;
 				this.debugMessage( '   Shininess : ' + shininess );
 
+			} else if ( next === MAT_TRANSPARENCY ) {
```

Comment:
Did you test this PR?

Where is `MAT_TRANSPARENCY` defined?

Reply:
yes, i had tested, first i read code of lib3ds 
`case CHK_MAT_SHININESS: {
                lib3ds_chunk_read_reset(&c, io);
                int_percentage_read(&material->shininess, io);
                break;
            }

            case CHK_MAT_SHIN2PCT: {
                lib3ds_chunk_read_reset(&c, io);
                int_percentage_read(&material->shin_strength, io);
                break;
            }

            case CHK_MAT_TRANSPARENCY: {
                lib3ds_chunk_read_reset(&c, io);
                int_percentage_read(&material->transparency, io);
                break;
            }
`
then, There also has MAT_TRANSPARENCY parameter in TDSLoader

Reply:
`MAT_TRANSPARENCY` is undefined, so your code will throw an error.

Reply:
`
var MAT_TRANSPARENCY = 0xA050;
`
it is defined in TDSLoader.js  at line 965

Reply:
The line is commented out:
https://github.com/mrdoob/three.js/blob/c52fef3b548ecf7d252850ffaa9a2eaca6b26989/examples/js/loaders/TDSLoader.js#L921

Reply:
oh, the version of my local THREE.js is v101, that line didn't commented

Reply:
Better use the `dev` branch as base for your changes when submitting PRs.

Reply:
yes, it is > dev branch in my github account

Reply:
You just said that you used r101 as base for your PR. `dev` is on r109dev...

Reply:
sorry, maybe i didn't say very clearly, In my github account, i fllow your `dev` version, and at my local computer, i used 101 version.